src/FSharp.Core/array.fsi

```diff
@@ -3252,6 +3252,301 @@ module Array =
         [<Experimental("Experimental library feature, requires '--langversion:preview'")>]
         val tryPick: chooser: ('T -> 'U option) -> array: 'T[] -> 'U option
 
+        /// <summary>Applies a function to each element of the array in parallel, threading an accumulator argument
+        /// through the computation for each thread involved in the computation. After processing entire input, results from all threads are reduced together.
+        /// Raises ArgumentException if the array is empty.</summary>
+        /// <remarks>The order of processing is not guaranteed. For that reason, the 'reduce' function argument should be commutative.
+        /// (That is, changing the order of execution must not affect the result)
+        /// Also, compared the non-parallel version of Array.reduce, the 'reduce' function may be invoked more times due to the resulting reduction from participating threads.</remarks>
+        ///
+        /// <param name="reduction">The function to reduce a pair of elements to a single element.</param>
+        /// <param name="array">The input array.</param>
+        ///
+        /// <exception cref="T:System.ArgumentNullException">Thrown when the input array is null.</exception>
+        /// <exception cref="T:System.ArgumentException">Thrown when the input array is empty.</exception>
+        ///
+        /// <returns>The final result of the reductions.</returns>
```

Comment:
I don't really like this term, [the result](https://en.wiktionary.org/wiki/result) is already the final product. I'd prefer using just "result" here and "partial"/"chunk" results elsewhere. At least in the public doc.

Reply:
OK